spring/boot-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -196,6 +215,67 @@ ArmeriaServerConfigurator actuatorServerConfigurator(
         };
     }
 
+    @Bean
+    @ConditionalOnProperty("management.server.port")
+    ArmeriaServerConfigurator secureActuatorServerConfigurator(WebEndpointProperties properties,
+                                                               ManagementServerProperties serverProperties,
+                                                               ConfigurableEnvironment environment,
+                                                               ArmeriaSettings armeriaSettings) {
+        return sb -> {
+            final Port port = obtainManagementServerPort(serverProperties.getPort());
+            configurePorts(sb, ImmutableList.of(port));
+            addLocalManagementPortPropertyAlias(environment, port);
+            configureSecureDecorator(sb, port, properties.getBasePath(), armeriaSettings);
+        };
+    }
+
+    private static Port obtainManagementServerPort(Integer port) {
+        int actualPort = requireNonNull(port, "port");
+        if (actualPort == 0) {
+            actualPort = SocketUtils.findAvailableTcpPort();
+        }
+        return new Port().setPort(actualPort).setProtocol(SessionProtocol.HTTP);
```

Comment:
Question: Don't we need to support HTTPS? The upstream seems to support TLS/SSL configuration.
https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-management-specific-ssl

If that is worthy to handle, I think you can do it in the following PR. 😉

Reply:
Can I set up another certificate on a specific port in Armeria? I think this is a bit vague.

Anyway, I felt the need to deal with [this](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-disabling-http-endpoints).

Reply:
> Can I set up another certificate on a specific port in Armeria? I think this is a bit vague.

I thought that the actuator shares the Server's certification.
We cannot perfectly handle the feature, now. Please ignore this.

> Anyway, I felt the need to deal with this. WDYT?

That sounds good. 👍

Reply:
Thanks :)